app/views/shared/_flash_messages.erb

```diff
@@ -1,11 +1,32 @@
 <% if flash[:notice] %>
-  <div class="govuk-!-padding-top-6 govuk-!-padding-bottom-6">
-    <div class="govuk-panel govuk-panel--notice">
-      <div class="govuk-panel__body govuk-!-font-size-19 govuk-!-font-weight-bold">
-        <%= flash[:notice] %>
+  <% if flash[:notice] == 'first_time_login' %>
+    <h1>Vitajte na Návody.Digital</h1>
+    <div class="govuk-modal-dialogue govuk-modal-dialogue--open" data-module="govuk-modal-dialogue" data-controller="modal" data-modal-target="modal">
+      <div class="govuk-modal-dialogue__wrapper">
+        <dialog aria-describedby="welcome-description" aria-labelledby="welcome-title" class="govuk-modal-dialogue__box" id="welcome" role="alertdialog" tabindex="0" open="">
+          <div class="govuk-modal-dialogue__header">
+            <button aria-label="Close modal dialogue" class="govuk-button govuk-modal-dialogue__close" data-action="modal#close" data-module="govuk-button" type="button">×</button>
+          </div>
+          <div class="govuk-modal-dialogue__content">
+            <h2 class="govuk-heading-l" id="welcome-title">Vitajte na Návody.Digital 👋</h2>
+            <div class="govuk-modal-dialogue__description" id="welcome-description">
+              <p class="govuk-body-l">Kedže ste sa prihlásili prvý krát, odporúčame, aby ste si pozreli krátky návod, v
```

Comment:
slova `prihlásili prvý krát` a `pozreli krátky návod` by som dal boldom. to ti oko automaticky prescanuje a hned vies co sa ta to pyta. a ze to nie je nejaky spam, ktory mas tendenciu hned zavriet.

Reply:
![image](https://github.com/slovensko-digital/navody.digital/assets/26342/96a1695b-dd98-4f4f-9f3f-f645e88bc22e)